res/src/onissue.py

```diff
@@ -0,0 +1,54 @@
+import os 
+
+content = os.environ['ISSUE_INPUT'] 
+	
+# read paths and files
+with open("res/config.txt") as f:
+	for line in f:
+		line = str((line.strip()))
+		if line.find("indexfile") == 0:	# i.e. indexfile = README.md | can get changed to whatever file you want the script output go
+			indexfile = line.split(" = ")[1]
+		if line.find("template") == 0:	# i.e. template = res/template.txt | a template how the indexfile should look, can be edited
+			template = line.split(" = ")[1]
+		if line.find("listfolder") == 0:	# i.e. listfolder = res/pluginlist/ | folder to where the plugin descriptions files are
+			listfolder = line.split(" = ")[1]
+		if line.find("pathtoplugins") == 0:	# i.e. pathtoplugins = myplugins/ | folder where the plugin folder and files are
+			pathtoplugins = line.split(" = ")[1]
+		if line.find("webroot") == 0:	# i.e. webroot = https://github.com/zuckung/test3/blob/main/
+			webroot = line.split(" = ")[1]	
+		if line.find("pluginurl") == 0:	# i.e. pluginurl = https://github.com/zuckung/test3/tree/main/Working/All%20Plugins/
+			pluginurl = line.split(" = ")[1]
+		if line.find("assetfullpath") == 0:	# i.e. assetfullpath = https://github.com/zuckung/test3/releases/download/Latest/
+			assetfullpath = line.split(" = ")[1]
+		if line.find("newsamount") == 0:	# i.e. newsamount = 20
+			newsamount = line.split(" = ")[1]
+
+def format_content():
+	splitted = content.split('\n')
+	if splitted[0] == '### Pluginname\r':
```

Comment:
```suggestion
	if splitted[0] == '### Name\r':
```
I changed this in the template to be just "Name", not "Pluginname"